Makefile

```
@@ -133,6 +134,14 @@ squash-root: dep
 	@rm -f $(INITRAMFS_DIR)/rootfs.tar
 	@echo "  -> $(INITRAMFS_DIR)/rootfs.tar.gz"
 
+docker-image: squash-root
```

checkmake (0.3.2)

[warning] Target body for "docker-image" exceeds allowed length of 5 lines (6).

(maxbodylength)